journal/updated/01-journal.adoc

```diff
@@ -1031,237 +1031,250 @@ and the high-sheriff continued loving and kind to Friends,
 towards the aldermen that interceded for me;
 for the captain knew I was a peaceable man,
-and never concerned myself in fighting for one side or other.
+and never concerned myself in fighting for one side or the other.
 When the troop had their orders, they went on their way; and I praised God,
-in the multitude of his mercies, that there was no blood shed that day;
-for many of the young men of the town, with the aldermen,
-were gathered together with clubs and staves, saying,
-"`What, should a town`'s born child be so abused by such a bad fellow as that was,
-before-mentioned?`"
-My heart often trembled within me, lest anything should fall amiss in
-this tumult; and I desired them often, before they went to the captain,
-that I might go along with them towards my prison.
+in the multitude of His mercies, that there was no blood shed that day;
+for many of the young men of the town, as well as the aldermen,
+were gathered together with clubs and staffs, saying,
+"`What! Should a child born in this town be so abused by such
+a bad fellow?`" My heart at various times trembled within me,
+lest any violence should be committed in
+this tumult; and I requested of them many times,
+before they went to speak with the captain,
+that they might let me go along with the troop to prison.
 
 So that night I stayed at home,
-and next morning took my journey towards my prison at Montgomery.
-I avoided the house of my uncle, a justice of the peace in this county, near my way,
-and brother-in-law to this captain, lest he should stop me from going to prison.
-
-So I went there myself alone, and told the marshal, John Mason,
-that I was come a prisoner; and he took me up to an upper garret for my lodging,
-but I had the liberty of the house, as well as other prisoners,
+and the next morning made my journey towards the prison at Montgomery.
+On my way I avoided the house of my uncle, a justice of the peace in this county,
+and brother-in-law to the captain before-mentioned,
+lest he should stop me from going to prison.
+
+So I went there alone, and told the marshal, John Mason,
+that I had come as prisoner; and he took me up to an upper garret for my lodging.
+But I had freedom of the whole house, as did the other prisoners,
 there being many Presbyterians, Independents, and Baptists,
-who were formerly my great acquaintance; but now they appeared very strange,
+who were formerly very acquainted with me; but now they appeared very strange,
 and would not discourse with me.
-I considered the reason, and was informed,
-that these old formal church-members or professors had agreed
+I wondered at the reason for this, and was informed
+that these old, formal church-members or professors had agreed
 among themselves that they would not discourse with me,
 nor receive any books from me,
-lest the most serious inward Christians among them should turn Quakers.
+lest the most inwardly serious Christians among them turn Quakers.
```

Comment:
I think we should leave this the way it was, but add the word "and". LIke this: lest the most serious **and** inward Christians among them should turn Quakers.